public/index.html

```diff
@@ -3,10 +3,15 @@
   <head>
     <title>CU-News</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Righteous" rel="stylesheet">
   </head>
 
   <body>
-    <h1 style="text-align: center">CU-News</h1>
+    <h1 style="text-align: center;
```

Comment:
we are just styling a headline in the html file. it's just 1 tiny thing. IF we want to style the whole application, then i think we should make a css file.